ztag/transforms/http.py

```diff
@@ -45,6 +104,7 @@ def _transform_object(self, obj):
                         title = title[0:1024]
                     out['title'] = title.strip()
             if headers:
+                # FIXME: This modifies the input?
```

Comment:
WAT. might want to add a note in a comment up at the function header.

Reply:
Yeah, good call

Reply:
oh, dear